spec/models/post_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe Post, type: :model do
+  describe "Associations" do
+    context 'generic post' do
+      it "should belong to a user" do
+        subject do
+          post = build(:post)
+        end
```

Comment:
Does this pass? The setup blocks are used outside of the it block, so this would look something like:
```ruby
describe "Associations" do
  subject { build(:post) }

  it { should belong_to(:user) }
end
```